<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Material</class>
 <widget class="QWidget" name="Material">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material properties</string>
  </property>
  <widget class="QPushButton" name="pb_OK">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_alpha">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>268</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Resistivity [nOhm m]:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>criogenic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ELECTROMAGNETIC PROPERTIES:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_rho_rt">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_E_cr">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>268</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear elastic modulus [MPa]:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pr_rt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>268</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mass density [kg/m^3]:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_E_rt">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>room temperature (20°C)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>criogenic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>room temperature (20°C)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>230</y>
     <width>268</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Thermal expansion coefficient:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>268</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear elastic modulus [GPa]:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Y">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_rho_cr">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pr_cr">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MECHANICAL PROPERTIES:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_density">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>THERMAL PROPERTIES:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>268</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Yeld limit [MPa]:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>room temperature (20°C)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>130</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>criogenic</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_material">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Available materials:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_del">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_import">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
